#class MyException(Exception):
    #pass
#try:
    #raise MyException
#except:
    #print("Custom Error Occurred")
    
    
is_logged_in = True
requires_login = not is_logged_in
print(requires_login)